spp_rest_auth_jwt/models/auth_jwt_validator.py

```diff
@@ -228,3 +229,15 @@ def write(self, vals):
     def unlink(self):
         self._unregister_auth_method()
         return super().unlink()
+
+    def copy(self, default=None):
+        name = f"{self.name}_{uuid.uuid4().hex}"
+        if default:
+            default.update({"name": name})
+        else:
+            default = {"name": name}
+
+        self.ensure_one()
```

Comment:
we shouldn't call `ensure_one` here since the base models method already did that.

Reply:
Hi @nhatnm0612 , done removing the ensure_one in the function. Thanks.